src/app/adminka/components/active.tsx

```
@@ -0,0 +1,324 @@
+          return {
+            // url: `/markets/clob/propose/batch-resolve`,
+            url: 'markets/clob/propose/batch-resolve',
+            payload: indexes,
+          }
+        case 'group':
+          return {
+            // url: `/markets/group//propose/resolve`,
+            url: `markets/group/${selectedMarkets[0].id}/propose/resolve/execute`,
+            payload: indexes,
+          }
+        case 'amm':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)